doc/doxygen/src/roadmap.md

```diff
@@ -9,85 +9,66 @@ The text and items below are tentative, up for discussion, to be updated by regu
 (contact/steering: [Martine](https://github.com/miri64))
 
 1. ICN stack support clean-up
-2. discuss mid- and long-term plans for network stack maintenance & development (GNRC vs other supported stacks)
-3. Support for CoAP blockwise transfer, observe. Also see [Ken's](https://github.com/kb2ma) unofficial [CoAP for RIOT](https://github.com/kb2ma/RIOT/wiki/CoAP-Home) page.
-4. revisit network time synchronization
+1. discuss mid- and long-term plans for network stack maintenance & development (GNRC vs other supported stacks)
+1. revisit network time synchronization
```

Comment:
Sadly, just using `1.` does not work with Doxygen: https://ci.riot-os.org/results/528163af52e846bda1f4ff84f035d18a/doc-preview/roadmap.html

Reply:
Same with a more recent version of Doxygen. Let's go for an unnumbered list then; keeping list numbers in sync is nothing humans should do.

Reply:
:+1: I am not even sure numbered lists make sense here. It gives the impression that there is a priority (which IMHO we don't have and can't provide)